hack/util.sh

```diff
@@ -641,7 +641,11 @@ function install_registry {
 }
 
 function wait_for_registry {
-	wait_for_command '[[ "$(oc get endpoints docker-registry --output-version=v1 --template="{{ if .subsets }}{{ len .subsets }}{{ else }}0{{ end }}" --config=${ADMIN_KUBECONFIG} || echo "0")" != "0" ]]' $((5*TIME_MIN))
+	local config="--config=${ADMIN_KUBECONFIG}"
+	local tmplsubsets='{{ if .subsets }}{{ len .subsets }}{{ else }}0{{ end }}'
```

Comment:
I've been holding off on this for a while, but ...

If we do a simple "string is equal to a non-zero number" check, we don't need the `if/else` in this template or the `|| echo 0` in the next command. 

I also would like to see us transition to `jsonpath` as it seems easier to understand and write 99% of the time. However, we can't do `len()` in `jsonpath` so the question becomes ... are we always happy to see  that the service has *any* endpoints? Do we need / can we use a less fragile check? Since we're doing the setup in a function, we can assume things about which endpoints should exist?